manager/state/raft/raft.go

```diff
@@ -394,6 +397,16 @@ func (n *Node) Run(ctx context.Context) error {
 				}
 			}
 
+			n.once.Do(func() {
+				// If we are the only registered member after restoring
+				// from the state, campaign to be the leader.
+				if len(n.cluster.Members()) <= 1 {
+					if err := n.Campaign(n.Ctx); err != nil {
```

Comment:
On `swarm init` we would end up calling `Campaign` twice (there is another call in `JoinAndStart`). Should we remove one or the other?